KEEP/etc/rc.local

```diff
@@ -20,6 +20,7 @@ do_disable_ipv6_accept_ra=true
 do_cleanup_x11=true
 do_user_cpufreq=false
 do_all_ports_unprivileged=false
+do_fuse=false
```

Comment:
i guess we should name this do_user_fuse so its a little more descriptive, and along the lines of do_user_cpufreq. and maybe even move the entry one up so they're grouped together.

Reply:
also the commit message is somewhat misleading and too long; i guess "rc.local: add template for /dev/fuse permissions" or something like that would be better